app/src/main/java/com/projectit3048c/dto/FoodNutrients.kt

```diff
@@ -1,3 +1,2 @@
 package com.projectit3048c.dto
-
-data class FoodNutrients(var nutrientId: Int = 0, var nutrientName: String = "nothing", var nutrientNumber: Double = 0.0, var value: Double = 0.0)
+data class FoodNutrients(var nutrientId: Int = 0, var nutrientName: String = "", var nutrientNumber: Double = 0.0, var value: Double = 0.0)
```

Comment:
Removed nothing because it represents a value that cannot exist. Replacing it with an empty string allows an option to change values unless it never changes.